lib/clientv2/workloadendpoint_e2e_test.go

```diff
@@ -38,21 +38,34 @@ var _ = testutils.E2eDatastoreDescribe("WorkloadEndpoint tests", testutils.Datas
 	ctx := context.Background()
 	namespace1 := "namespace-1"
 	namespace2 := "namespace-2"
-	name1 := "wep-1"
-	name2 := "wep-2"
-	spec1 := apiv2.WorkloadEndpointSpec{
+	name1 := "node--1-k8s-abcdef-eth0"
```

Comment:
btw, I changed these as a future fix for when we add validation - since the name and the spec are inextricably linked I couldn't just swap the same Spec between different resource instances.  didn't need to be done now, but there you go.